skypy/pipeline/_pipeline.py

```diff
@@ -126,8 +128,6 @@ def __init__(self, configuration):
         # - keep track where functions need to be called
         #   functions are tuples (function name, [function args])
         functions = {}
-        for job in self.parameters:
-            self.dag.add_node(job)
```

Comment:
We do need the nodes in the DAG though, or the check in line 164 will fail for anything that depends on a parameter. Same with the `cosmology` node.